src/modules/auth/payloads/register.payload.ts

```diff
@@ -1,37 +1,25 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsEmail, IsNotEmpty, MinLength } from 'class-validator';
-import { SameAs } from '../../common/validator/same-as.validator';
+import { IsEmail, IsNotEmpty } from 'class-validator';
 
 export class RegisterPayload {
   @ApiProperty({
     required: true,
     example: 'KimAng',
   })
   @IsNotEmpty()
-  firstname: string;
+  public firstname: string;
 
   @ApiProperty({
     required: true,
     example: 'Kheang',
   })
   @IsNotEmpty()
-  lastname: string;
+  public lastname: string;
 
   @ApiProperty({
     required: true,
     example: 'kimangkheang@gmail.com',
   })
   @IsEmail()
-  email: string;
-
-  @ApiProperty({
-    required: true,
-  })
-  @IsNotEmpty()
-  @MinLength(5)
-  password: string;
-
-  @ApiProperty({ required: true })
-  @SameAs('password')
-  passwordConfirmation: string;
+  public email: string;
```

Comment:
Maybe no need to expose personal email.
Also, the field is public by default!
```suggestion
  @ApiProperty({
    required: true
  })
  @IsEmail()
  email: string;
```